3rdparty/README.md

```diff
@@ -0,0 +1,32 @@
+### Quantizer Utilities
```

Comment:
i'm confused on why do we need to vendor the script? is this the same script from ammo or written by you?

Reply:
@simon-mo , this is a script taken from nVIDIA, as part of their quantizer examples, we kept them under `3rdparty`, and their license banner.

Reply:
If there's no change, let's not include it please. We are should comfortable referring users to the ammo/quantizer repo to perform the quantization.